packages/teams-js/src/internal/constants.ts

```diff
@@ -175,3 +175,12 @@ export const teamsDeepLinkProtocol = 'https';
  * Limited to Microsoft-internal use
  */
 export const teamsDeepLinkHost = 'teams.microsoft.com';
+
+/**
+ * @hidden
+ * Error thrown when the {@linkcode app.initialize} has not yet successfully completed
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export const errorLibraryNotInitialized: Error = new Error('The library has not yet been initialized');
```

Comment:
I hope not because we do something very similar for errorNotSupportedOnPlatform, but I will verify

Reply:
If it is a problem maybe you could just make this a function that throws and then it can be called in the context of the callstack that needs to trigger it?

Reply:
Alright, after playing with this for a while marcin was right that this messed up our callstacks. It seemed like the best way to fix it was to make the error string a constant and then have everyone who wants to use it make a new Error and pass this string to it. That makes sure all of the strings are using the same shared constant.

Note: I think this isn't an issue for errorNotSupportedOnPlatform because that's an SdkError instead of an Error, which doesn't store callstack information